scio-bigquery/src/it/scala/com/spotify/scio/bigquery/BigQueryClientIT.scala

```diff
@@ -39,13 +39,11 @@ class BigQueryClientIT extends FlatSpec with Matchers {
 
   "extractLocation" should "work with legacy syntax" in {
     val query = "SELECT word FROM [data-integration-test:samples_%s.shakespeare]"
-    bq.extractLocation(query.format("us")) shouldBe Some("US")
```

Comment:
i removed the US test cases from `extractLocation` -- i would expect that any regressions would be caught by the remaining EU test case. what do you think?

Reply:
yup 👍